index.html

```diff
@@ -9,63 +9,66 @@
-                excepturi assumenda.
-                Rerum unde quidem maxime. Ullam, dolores labore molestias voluptatibus adipisci quod? Quaerat nihil
-                inventore molestiae assumenda, voluptates vel necessitatibus eligendi? Accusamus ratione eligendi
-                reprehenderit? Beatae incidunt molestias ea soluta nulla.
-                Fugiat quae soluta voluptate mollitia praesentium modi ex illo, autem assumenda, vel aliquam fugit
-                possimus dolorem esse corporis ipsa! Voluptatem sapiente error non, ea optio suscipit quasi maxime nobis
-                ex?
-                Et ipsa tenetur ullam ab nesciunt quas fugit velit impedit illum esse eligendi corporis culpa
-                praesentium aspernatur ex sint veniam quidem, doloribus corrupti hic vitae. Eum neque ducimus omnis
-                exercitationem?
-            </p>
+        <section class="top">
+            <div class="left">
+                <form>
+                    <label id="question-form">
+                        Question
+                        <input name="input-question">
+                    </label>
+                    <!-- <div class="option-input"> -->
+                        <label>
+                            Option A
+                            <input name="input-option-a">
+                        </label>
+                        <label>
+                            Option B
+                            <input name="input-option-b">
+                        </label>
+                    <!-- </div> -->
+                </form>
+                <button id="submit-button">Submit</button>
+            </div>
+            <div class="right">
+                <div class="current-poll">
+                    <h3>Current Poll</h3>
+                    <p id="poll-question"></p>
+                    <div class="options">
+                        <div id="option-a">
+                            <p id="option-a-title"></p>
+                            <p id="option-a-votes"></p>
+                        </div>
+                        <div id="option-b">
+                            <p id="option-b-title"></p>
+                            <p id="option-b-votes"></p>
+                        </div>
+                    </div>
+                </div>
+                <div class="count-buttons">
+                    <div class="add">
+                        <button id="option-a-add">+</button>
+                        <button id="option-b-add">+</button>
+                    </div>
+                    <div class="sub">
+                        <button id="option-a-sub">-</button>
+                        <button id="option-b-sub">-</button>
+                    </div>
+                </div>
+                <button id="close-poll">Close Poll</button>
+            </div>
+        </section>
+        <section class="bottom">
+            <h3>Past Polls</h3>
+            <div>
```

Comment:
This is where I would create that div with a class `post-polls` as a destination for your DOM node that you create with `renderPastPolls`